README.md

```diff
@@ -52,8 +54,16 @@ deobfuscates a dalvik executable
  -v,--verbose <LEVEL>            Set verbosity to LEVEL, default: 0
 ```
 
+NOTES: 
+• simplify.jar must be built first. See the next section for more informations.
+
+• Name of the jar may vary. After the build check the simplify/build/libs folder, copy its path and the jar name and paste the entiere path (under quotes) as executable instead of simplify.jar if java cannot locate it itself.
+
+
 ## Building
 
+**Linux/Mac users**
+
 Because this project contains submodules for Android frameworks, either clone with `--recursive`:
```

Comment:
The cloning steps aren't unique to Linux / Mac users.

Reply:
Really? Since it wasn't needed for Windows I thought so...

Reply:
If you clone the repo using git without using `--recursive`, the smali framework submodule will not be downloaded. I think the build will fail, the tests will certainly fail, and if a jar is produced it won't work.

Reply:
For Linux and Mac users, so. I didn't perform such things under Windows (simply because I can't) and everything worked well. Or maybe I did not understand correctly, again.

Reply:
How did you download the project?
What happened when you ran simplify.jar against something?

I'm certain that if you don't download the giant pile of smali files for the Android operating system and include them in the smalivm / simplify jar, shit will be very broken. The build may succeed, but nothing will work.

Reply:
Well, I've downloaded the whole simplify fork from the repository. Then I just ran `gradlew fatjar` and tested simplify-1.2.0.jar on a jar I wanted to clean. Worked fine. I think it worked fine, no errors but it didn't clear anything, maybe because the jar of the app I tested wasn't so obfuscated.

Reply:
What's the size of simplify-1.2.0.jar?

You ran it against a jar, but simplify only works on apk and dex files, or at the very least dalvik bytecode. I'm pretty sure your jar isn't correct and simplify didn't do anything, so the brokenness wasn't exposed.

Reply:
simplify-1.2.0.jar is 21 841Ko large.
The jar I ran simplify against was converted from dex itself. Then after the simplifying process it output the cleaned mess as a dex file.